Example/Backpack/BPKAppDelegate.m

```diff
@@ -33,8 +35,15 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
 
     ShakeableWindow *shakeableWindow = [[ShakeableWindow alloc] initWithFrame:[[UIScreen mainScreen] bounds]];
     shakeableWindow.windowLevel = UIWindowLevelAlert;
-    shakeableWindow.rootViewController = self.window.rootViewController;
-    self.shakeableWindow = shakeableWindow;
+    [BPKLondonTheme apply];
```

Comment:
Unrelated, but I wonder if the wording `apply` is misleading. Maybe the API should be `[BPKLondonTheme enable];`?

Reply:
I think we should probably rethink this API a bit, but I'd suggest we do that in another PR. Having both `BPKXTheme` and `BPKXThemeCotainer` seems non-ideal to me. My thoughts right now is that it would be nicer if `BPKXTheme` had a method that returned an instance of a theme container for that theme. This can also be formalized in a protocol so consumers can define their own themes in a predictable way